cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -26,15 +26,15 @@ const (
 
 // GetKubeDNSVersion returns the right kube-dns version for a specific k8s version
 func GetKubeDNSVersion(kubeVersion *version.Version) string {
-	// v1.7.0+ uses 1.14.4, just return that here
+	// v1.8.0+ uses 1.14.4, just return that here
```

Comment:
comment doesn't match code

Reply:
@liggitt Here is one follow-up [PR](https://github.com/kubernetes/kubernetes/pull/52501) for code.